index.rst

```diff
@@ -297,6 +297,7 @@ Cookbook
     BME280 Environment, cookbook/bme280_environment, bme280.jpg
     Sonoff Fishpond Pump, cookbook/sonoff-fishpond-pump, cookbook-sonoff-fishpond-pump.jpg
     H801 LED Controller, cookbook/h801, h801.jpg
+    SSD1306 I2C Guide, cookbook/oled, oled.png
```

Comment:
The oled.png image is too big, 158kb is too much for the main index - see contributing guide for how to shrink it down.